doc/api/child_process.md

```diff
@@ -887,26 +925,28 @@ In the following example, only the child's fd `1` (stdout) is configured as a
 pipe, so only the parent's `child.stdio[1]` is a stream, all other values in
 the array are `null`.
 
-    const assert = require('assert');
-    const fs = require('fs');
-    const child_process = require('child_process');
+```js
+const assert = require('assert');
+const fs = require('fs');
+const child_process = require('child_process');
 
-    const child = child_process.spawn('ls', {
-        stdio: [
-          0, // Use parents stdin for child
-          'pipe', // Pipe child's stdout to parent
-          fs.openSync('err.out', 'w') // Direct child's stderr to a file
-        ]
-    });
+const child = child_process.spawn('ls', {
+    stdio: [
+      0, // Use parents stdin for child
+      'pipe', // Pipe child's stdout to parent
+      fs.openSync('err.out', 'w') // Direct child's stderr to a file
+    ]
+});
```

Comment:
Since there's churn here anyway, maybe make the indentation consistent while you're at it?